packages/grafana-data/src/types/datasource.ts

```diff
@@ -555,7 +555,7 @@ export interface DataQueryRequest<TQuery extends DataQuery = DataQuery> {
   rangeRaw?: RawTimeRange;
   timeInfo?: string; // The query time description (blue text in the upper right)
   panelId?: number;
-  panelPluginType?: string;
```

Comment:
Can you follow the deprecation process to remove fields from types?

Reply:
@academo this has only existed in main for 3 weeks

Reply:
@academo March 5th is the 10.4 release which is why I am backporting this so we do not release a version of grafana/data with this prop name

Reply:
@torkelo thanks for double checking!